prusti-viper/src/encoder/high/lower/expression.rs

```diff
@@ -1,7 +1,8 @@
 use super::super::types::interface::HighTypeEncoderInterfacePrivate;
 
 use super::IntoPolymorphic;
-use vir_crate::{high as vir_high, polymorphic as vir_poly};
+use vir_crate::{high as vir_high, legacy::FloatConst, polymorphic as vir_poly};
```

Comment:
```suggestion
use vir_crate::{high as vir_high, legacy as vir_low, polymorphic as vir_poly};
```
and use `vir_low::FloatConst`